services/forms-api/lambdas/updateQuestion.js

```diff
@@ -0,0 +1,38 @@
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as response from '../../../libs/response';
+import { updateItem } from '../helpers/queries';
+
+export async function main(event) {
+  const { formId, stepId, questionId } = event.pathParameters;
+  const questionPartitionKey = `FORM#${formId}`;
+  const questionSortKey = `${questionPartitionKey}#STEP#${stepId}#QUESTION#${questionId}`;
+
+  const requestBody = JSON.parse(event.body);
+
+  const validKeys = ['label', 'description', 'show', 'type'];
```

Comment:
For the questions, we want to allow any key, because different types of inputs requires different additional props. So here any kind of key (possibly excluding some, like PK and SK) should be accepted and updated.